examples/getting-started-movielens/03-Training-with-PyTorch.ipynb

```diff
@@ -37,52 +37,44 @@
     "\n",
```

Comment:
<p>we do not have <code>merlin-pytorch-training</code> container. Can you please rephrase it as <code>merlin-pytorch</code> ?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/Merlin/pull/779/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
fixed (and fixed the link)